src/nanosvg.h

```diff
@@ -111,55 +111,52 @@ typedef struct NSVGgradientStop {
 } NSVGgradientStop;
 
 typedef struct NSVGgradient {
+	NSVGgradientStop stops[1];
```

Comment:
THis change will break any gradient with more than one stop. (IOW, all of them). It's at the end of the struct so the array can be made larger simply by allocating more bytes for the struct. See https://github.com/memononen/nanosvg/blob/master/src/nanosvg.h#L844

Reply:
Hm, you are right, thanks, I've missed that part. 
Great coding style.... 
Sorry for disturbing.